_layouts/docs.html

```diff
@@ -28,12 +28,7 @@ <h1>{{ page.title }}</h1>
                             {%- if page.advisory -%}
                             <blockquote>{{ site.data.advisories.texts[page.advisory] | markdownify }}</blockquote>
                             {%- endif -%}
-                            {%- unless page.skip_read_time == true -%}
-                            {%- assign words = content | number_of_words -%}
-                            {%- if words >= 360 -%}
-                            <p><em class="reading-time">Estimated reading time: {{ words | divided_by:180 }} minutes</em></p>
-                            {%- endif -%}
-                            {%- endunless -%}
+                            <p>{%- include read_time.html -%}</p>
```

Comment:
I guess alternatively, we could add a style for block quotes immediately following a H1;

```css
h1+blockquote {
  margin-top: 24px;
}
```

Reply:
Yes but looks like the `<blockquote>` is conditional?: https://github.com/docker/docker.github.io/blob/72a3a45c0e09a7c101d66dca13c7fae3916c96f3/_layouts/docs.html#L28

Maybe add a new css class to https://github.com/docker/docker.github.io/blob/72a3a45c0e09a7c101d66dca13c7fae3916c96f3/_layouts/docs.html#L26